Frends.AmazonS3.ListObjects/Frends.AmazonS3.ListObjects/Definitions/Regions.cs

```diff
@@ -0,0 +1,35 @@
+﻿
+namespace Frends.AmazonS3.ListObjects.Definitions
+{
+    /// <summary>
+    /// AWS regions.
+    /// </summary>
+    public enum Regions
```

Comment:
Enum itself is only one region, not multiple regions, so the name should reflect that.